src/crypto/test/crypto.cpp

```diff
@@ -190,6 +195,107 @@ ccf::crypto::Pem generate_self_signed_cert(
     kp, name, {}, valid_from, certificate_validity_period_days);
 }
 
+std::string qcbor_buf_to_string(const UsefulBufC& buf)
```

Comment:
Suggest a to_uint8_vev() also, to avoid memcpy() above.

Reply:
Sorry, what is to_uint8_vev?

Reply:
Also, this's a test code, you sure we want any string copy optimisation here?

Reply:
I meant the memcpy further up in the PR. It's not the optimisation, it's the safety I am interested in :)

Reply:
Ok, so it's same thing we discussed in another comment, got it.